src/OrchardCore/OrchardCore.Users.Abstractions/Handlers/IExternalLoginUserHandler.cs

```diff
@@ -1,9 +1,9 @@
 namespace OrchardCore.Users.Handlers;
 
 /// <summary>
-/// Contract for external login events, this occurs when a tenant is set up.
+/// Contract for external login user operations.
 /// </summary>
-public interface IExternalLoginEventHandler
+public interface IExternalLoginUserHandler
```

Comment:
We can leave the old interface but inherit it from the new interface. Obsolete the old interface to give users transition path. Move all the method into the new interface so that the old interface has nothing in it. This approach may allow us to rename the interface, and allow us to backport it without breaking anything. 


But, we have to undo the class renaming so we can safely backport it. (In another PR you can rename the class if you want for v3 or absolute it as well using the same strategy as the interface.)